src/lib/alpha.js

```diff
@@ -42,6 +43,7 @@ export const alphanumeric = {
   'el-GR': /^[0-9Α-ω]+$/i,
   'es-ES': /^[0-9A-ZÁÉÍÑÓÚÜ]+$/i,
   'fr-FR': /^[0-9A-ZÀÂÆÇÉÈÊËÏÎÔŒÙÛÜŸ]+$/i,
+  'fr-CA': /^[0-9A-ZÀÂÆÇÉÈÊËÏÎÔŒÙÛÜŸ]+$/i,
```

Comment:
Same here, doing `alphanumeric['fr-CA'] = alphanumeric['fr-FR']` would be easier/better since it's the same regex

Reply:
Of course²